client/extensions/woocommerce/index.js

```diff
@@ -194,11 +195,11 @@ function getAnalyticsPath( path, params ) {
 	}
 
 	if ( '/store/settings/email/:site/:setup?' === path ) {
-		return !! params.setup ? '/store/settings/email/:site/:setup' : '/store/settings/email/:site';
+		return params.setup ? '/store/settings/email/:site/:setup' : '/store/settings/email/:site';
```

Comment:
I like the removal of the `!!`, as that feels overly verbose. I'm just wondering if there is any scenario you can think of where it would make a difference? I can't think of any.

Reply:
This was caught by JS linter actually.  I removed it as I could not think of any other scenarios. I think it's safe.

Reply:
Agreed.